mingw-w64-openssl/PKGBUILD

```diff
@@ -61,7 +68,13 @@ prepare() {
   apply_patch_with_msg \
      openssl-1.1.1-relocation.patch \
      0001-tests-work-around-the-MSYS-vs-Windows-paths-problem.patch \
-     0001-shlib_wrap.sh-accommodate-for-MINGW-builds-being-tes.patch
+     0001-shlib_wrap.sh-accommodate-for-MINGW-builds-being-tes.patch \
+     openssl-1.1.1-mingw-arm.patch
+
+  # Fix EC code being broken with Clang 14
+  # https://github.com/openssl/openssl/pull/18258
+  apply_patch_with_msg \
+     18258.patch
```

Comment:
This fix was merged and released in 1.1.1r, but that version [was withdrawn on Oct 12th](https://www.openssl.org/news/newslog.html). The next version will include this fix, so then this patch can be removed.